cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -130,6 +131,32 @@ def test_service_ansible_playbook_crud(ansible_repository):
     assert not cat_item.exists
 
 
+def test_service_ansible_playbook_tagvis(wait_for_ansible):
+    repositories = RepositoryCollection()
+    repository = repositories.create(
+        fauxfactory.gen_alpha(),
+        "https://github.com/quarckster/ansible_playbooks",
+        description=fauxfactory.gen_alpha()
+    )
+    cat_item = AnsiblePlaybookCatalogItem(
+        fauxfactory.gen_alphanumeric(),
+        fauxfactory.gen_alphanumeric(),
+        provisioning={
+            "repository": repository.name,
+            "playbook": "dump_all_variables.yml",
+            "machine_credential": "CFME Default Credential",
+            "create_new": True,
+            "provisioning_dialog_name": fauxfactory.gen_alphanumeric()
+        }
+    )
+    cat_item.create()
+    view = navigate_to(cat_item, 'Details')
+    view.policy.item_select("Edit Tags")
+    tag = ('Department', 'Support')
+    mixins.add_tag(tag)
+    assert view.smart_management.get_text_of("My Company Tags") == mixins.get_tags()[0]
```

Comment:
**REQUIRED**
Please use the fixtures for the repository and the catalog item, do not duplicate the code. Also tags should be assigned via some method of `AnsiblePlaybookCatalogItem` instance.